app/services/import/persist/persist-company.service.js

```diff
@@ -0,0 +1,140 @@
+'use strict'
+
+/**
+ * Creates or updates an imported company and its child entities that have been transformed and validated
+ * @module PersistCompanyService
+ */
+
+const AddressModel = require('../../../models/address.model.js')
+const CompanyModel = require('../../../models/company.model.js')
+const ContactModel = require('../../../models/contact.model.js')
+const { db } = require('../../../../db/db.js')
+
+/**
+ * Creates or updates an imported company and its child entities that have been transformed and validated
+ *
+ * @param trx
+ * @param updatedAt
+ * @param {object} transformedCompanies - An object representing a valid WRLS licence
+ *
+ * @returns {Promise<string>} - the licence id from WRLS
+ */
+async function go (trx, updatedAt, transformedCompanies) {
+  await _persistCompanies(trx, updatedAt, transformedCompanies)
```

Comment:
We can return this directly.

```suggestion
  return _persistCompanies(trx, updatedAt, transformedCompanies)
```

Reply:
It does not return anything ?